extensions/editor.js

```diff
@@ -10,6 +10,8 @@ import './extended-blocks/core-embed';
 import './extended-blocks/paid-blocks';
 import './shared/styles/slideshow-fix.scss';
 
+import './store';
```

Comment:
Maybe we should add a comment here, so anyone looking at this main file quickly understands what the store can be used for?

Reply:
can we combine different events there?
We should set `env: { jest: true },`...